package.json

```diff
@@ -10,22 +10,28 @@
     "xmlhttprequest": "^1.8.0"
   },
   "devDependencies": {
-    "eslint-config-probot": "^0.1.0",
+    "eslint-plugin-import": "^0.1.0",
```

Comment:
please run `npm install --save-dev eslint-plugin-import` to correctly install this line